packages/autocomplete-js/src/autocomplete.ts

```diff
@@ -144,9 +144,9 @@ export function autocomplete<TItem>({
       dropdown.classList.remove('aa-Dropdown--stalled');
     }
 
-    const sections = state.suggestions.map((suggestion) => {
-      const items = suggestion.items;
-      const source = suggestion.source as InternalAutocompleteSource<TItem>;
+    const sections = state.collections.map((collection) => {
```

Comment:
I wonder if getSources should be renamed to getCollections too (or is that not the same?)

Reply:
They're not the same, as explained in https://github.com/algolia/autocomplete.js/pull/349#discussion_r511940785.